src/bot.ts

```diff
@@ -111,12 +135,28 @@ export const robot = (app: Probot) => {
         }
 
         async function getTokenOwnerFromApi(token: string, apiUrl: string): Promise<string> {
-          const tokenOwnerResponse = await axios.get(`${apiUrl}/tokens/${token}?extract=owner`);
-          if (tokenOwnerResponse && tokenOwnerResponse.data) {
-            return tokenOwnerResponse.data;
+          try {
+            const tokenOwnerResponse = await axios.get(`${apiUrl}/tokens/${token}?extract=owner`);
+            if (tokenOwnerResponse && tokenOwnerResponse.data) {
+              return tokenOwnerResponse.data;
+            }
+
+            return '';
+          } catch (error) {
+            console.log(`Cannot find token owner on the 'tokens' endpoint. Will try the 'collections'`);
```

Comment:
For more clarity, you can add in log also the owner value

Reply:
Also, you can add logger instead of console.log ( if possible )

Reply:
probot logger adds additional unnecessary information, so I would leave as console logs. Regarding the addition of the owner, since I cannot find it, I cannot log it. I could log the token ID but it's obvious since the worker only checks a token at a time